assets/prototype/domain/eventEditor/context/TicketProvider/useTicketEditorId.ts

```diff
@@ -10,7 +10,7 @@ import { EditorId } from '../../../../application/ui/components/layout/editor-mo
  * @param {string} prefix
  * @return {string} editor id for ticket
  */
-const useTicketEditorId = (prefix: string = 'ticket-editor', ticketId: EntityId): EditorId => {
+const useTicketEditorId = (prefix = 'ticket-editor', ticketId: EntityId): EditorId => {
 	return prefix !== '' ? `${prefix}-${ticketId}` : `ticket-editor-${ticketId}`;
```

Comment:
The same fix as for: https://github.com/eventespresso/event-espresso-core/pull/2210/commits/b0449345b0bf7b9cdd5e4e8b9dbdce371d0910c7